src/hotspot/share/code/compiledIC.cpp

```diff
@@ -586,9 +586,6 @@ bool CompiledStaticCall::set_to_clean(bool in_use) {
   // Reset call site
   set_destination_mt_safe(resolve_call_stub());
 
-  // Do not reset stub here:  It is too expensive to call find_stub.
-  // Instead, rely on caller (nmethod::clear_inline_caches) to clear
-  // both the call and its stub.
```

Comment:
While at it, I noticed this comment, which appears to be out of date.

Reply:
I read that comment more as a warning, in case in the future someone wondered why we don't reset the stub here, and tried to add it.  So I would leave it.  I think it's still useful.

Reply:
ok, reverted.